Makefile

```diff
@@ -213,7 +213,7 @@ yarn-install: clean-all
 	yarn --ignore-engines
 
 lerna-bootstrap: yarn-install
-	yarn lerna bootstrap -- --ignore-engines
```

Comment:
Unrelated to this PR, but we should try to remove `--ignore-engines`

Reply:
Yeah, we should give it a try after we drop node 8 support.

Reply:
Do we rely on a dependency which doesn't work with node 6?

Reply:
Ah, I see. We have proved it on #10249.